### ISSUE TITLE
WCAG 2.2: Passwords ‖ Target Size (Minimum)

### ISSUE THREAD

**Issue by querkmachine** [alphagov/govuk-design-system#2868]
## What

Update our guidance on passwords to emphasise that any controls that change the visibility of the password (e.g. a toggle to show or hide the password in plain text) needs to be at least 24 by 24 pixels in size.

### Resources

- [Passwords guidance page](https://design-system.service.gov.uk/patterns/passwords/)
- [WCAG 2.2 criterion for 'Target Size (Minimum)'](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)

## Why

The new WCAG 2.2 Level AA criterion 'Target Size (Minimum)' says: 

> The size of the [target](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html#dfn-target) for [pointer inputs](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html#dfn-pointer-input) is at least 24 by 24 [CSS pixels](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html#dfn-css-pixel)

As we do not currently provide any code for show/hide toggles on password fields, services may choose to implement one themselves—for example, to avoid the user typing a password twice when creating an account, which could be a failure of the ['Redundant Entry'](https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry.html) criterion. 

To combat this, we may want to emphasise this guidance as a temporary solution until such a time that we've developed a Design System component for this use case.

## Who needs to work on this
- Content designer or technical writer

## Who needs to review this
- Accessibility specialist
- Content designer or technical writer

## Done when

- [ ] Guidance has been updated

**Comment by dav-idc** [comment]
This has been added to the ['WCAG 2.2 content updates' document](https://docs.google.com/document/d/1nIxIKgVUheC8zvVdqWIcqtVPCl7oTN4hv11R9rDwdvM/edit?usp=sharing).

**Comment by calvin-lau-sig7** [comment]
Closing issue – content has been drafted, reviewed and added to pull request #3090 as part of https://github.com/alphagov/govuk-design-system/issues/3276.
